server/dive_tasks/utils.py

```diff
@@ -110,23 +112,25 @@ def organize_folder_for_training(data_dir: Path, downloaded_groundtruth: Path):
     return groundtruth
 
 
-def get_video_filename(folderId: str, girder_client: GirderClient) -> Optional[str]:
+def download_source_media(
+    girder_client: GirderClient, folder: GirderModel, dest: Path
+) -> List[str]:
```

Comment:
I've been wanting to change this for a while.  Rather than implement media discover both on the server and in tasks, just use the server endpoint to get the same exact information and download it.